READMEs/c2d-flow.md

```diff
@@ -42,54 +42,19 @@ pip install numpy matplotlib
 
 ### Set envvars
 
-In the work console:
-```console
-#set private keys of two accounts
-export TEST_PRIVATE_KEY1=0x5d75837394b078ce97bc289fa8d75e21000573520bfa7784a9d28ccaae602bf8
-export TEST_PRIVATE_KEY2=0xef4b441145c1d0f3b4bc6d61d29f5c6e502359481152f869247c7a4244d45209
-
-#set the address file only for ganache
-export ADDRESS_FILE=~/.ocean/ocean-contracts/artifacts/address.json
-
-#set network URL
-export OCEAN_NETWORK_URL=http://127.0.0.1:8545
-```
+Set the required enviroment variables as described in [datatokens-flow](datatokens-flow.md):
+- [x] Setup : Set envvars
 
 ### Start Python
 
-In the work console:
-```console
-python
-```
+In your project folder (i.e. my_project from `Install the library` step) and in the work console where you set envvars, run the following:
 
-For the following steps, we use the Python console. Keep it open between steps.
+## 2. Alice publishes a Data NFT
```

Comment:
The top of the README has the following steps.

```1. Setup
2. Alice publishes data asset
3. Alice publishes algorithm
4. ...
```

These steps are correct. 

Here's what they mean:
- Step 2 "publish data asset" means: create ERC721 data NFT for "DATA" asset, then create ERC20 datatoken for "DATA" asset
- Step 3 "publish algorithm" means: create ERC721 data NFT for "ALG" asset, then create ERC20 datatoken for "ALG" asset

So, there need to be fixes to the README to properly accommodate this. Right now the details for step 2 & 3 in this PR are both about the data asset. It really should be all in step 2. They should both reuse datatokens-flow.md if possible. A suggestion: reuse them, then at the tail end of step 2, do the following to make it clear to the user that you've created "DATA" assets not "ALG" assets: ```DATA_datatoken = erc20_token``` (and create other `DATA` variables as needed too).

Reply:
This readme is going to change with https://github.com/oceanprotocol/ocean.py/pull/674, should we move this conversation to that PR if this branch is merged first?

Here I have introduced changes to [c2d-flow.md](https://github.com/oceanprotocol/ocean.py/pull/697/files/5791a48a758b0b6228196e64a925fee8d132375e#diff-b30c139cd97de56d4e61b367afaa7c0477f931bd2f7bc1d681efe72765655207) related to deprecating repeated steps and linking to the `dataflows-flow.md` when needed but this readme has problems that I have not fixed since there is another PR to fix it.

What do you think @trentmc @DMats?

Reply:
I agree that this PR should be concerned with repeatng steps only. All other changes to c2d-flow should be part of #674